src/frequenz/client/common/pagination/__init__.py

```diff
@@ -0,0 +1,25 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Common code and utilities for Frequenz API clients.
+
+TODO(cookiecutter): Add a more descriptive module description.
+"""
+
+
+# TODO(cookiecutter): Remove this function
+def delete_me(*, blow_up: bool = False) -> bool:
+    """Do stuff for demonstration purposes.
+
+    Args:
+        blow_up: If True, raise an exception.
+
+    Returns:
+        True if no exception was raised.
+
+    Raises:
+        RuntimeError: if blow_up is True.
+    """
+    if blow_up:
+        raise RuntimeError("This function should be removed!")
+    return True
```

Comment:
As the comment suggest, this should be removed, this is only a code example in the repo-config template, but it shouldn't be included in all `__init__.py` files.

```suggestion
```